.changeset/weak-steaks-dream.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": minor
+---
+
+Savebar across the Dashboard has been updated to match the overall UI more closely, providing a more consistent user experience.
```

Comment:
This change is visible to the external user and he does not know what is "savebar" - this is internal term.

_Now you can see the updated appearance of the bottom bar that holds save and back actions. This means functionality has not changed_